UnityProject/Assets/Scripts/Core/Lifecycle/Spawn.cs

```diff
@@ -48,6 +48,12 @@ public static ObjectPool _ObjectPool
 	/// </summary>
 	public static readonly float DefaultScatterRadius = 0.1f;
 
+	public static void Clean()
+	{
+		Logger.Log("Removed  " + CleanupUtil.RidDictionaryOfDeadElements(nameToSpawnablePrefab) + " elements from " + nameof(Spawn) + "." + nameof(nameToSpawnablePrefab), Category.MemoryCleanup);
```

Comment:
[Follow our standard to how we write logs ](https://unitystation.github.io/unitystation/contribution-guides/Development-Standards-Guide/#0-in-code-documentation-and-clarity)